lib/nfg_ui/components/elements/slat_item.rb

```diff
@@ -19,36 +23,56 @@ def slat_header
           options.fetch(:slat_header, nil)
         end
 
-        def caption
-          options.fetch(:caption, nil)
+        # Leverage custom tooltip implementation
+        # so that tooltips are connected to slat item text
+        # and not the slat item containing div (resulting in better UI)
+        def tooltip
+          options.fetch(:tooltip, nil)
         end
 
         def render
           content_tag(base_element, slat_item_html_options) do
             if slat_header
-              concat(content_tag(:h6, slat_header, class: 'display-4 slat-column-header'))
+              concat(content_tag(:h6, slat_header, class: 'display-4 slat-column-header', **tooltip_html_options))
```

Comment:
@farhajawed - for slat items, *when slat header* and tooltip is present in options hash, tooltip is applied to the slat header.

Reply:
tooltip for slat has been demonstrated: https://github.com/network-for-good/nfg_ui_display_app/pull/1/files#diff-f923c1a03c0395daee0ee0faec384d30e096a9d86311f29bb6d3f0a424ef79edR86-R108